elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -174,7 +170,8 @@ def get_bugs_sweep(runtime: Runtime, find_bugs_obj, brew_event, bug_tracker):
     attached_bugs = bug_tracker.filter_attached_bugs(bugs)
     if attached_bugs:
         attached_bug_ids = {b.id for b in attached_bugs}
-        logger.warning("The following bugs have been attached to advisories: %s", attached_bug_ids)
+        logger.info("These bugs are already attached to some advisory, excluding them: %s",
```

Comment:
This could be fairly - make it info instead of warning. Further enhancement could be to complain if said advisories are not OCP advisories